modules/cnf-installing-numa-resources-operator.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-topology-aware-scheduling.adoc
+
+[id="installing-the-numa-resources-operator_{context}"]
+= Installing the NUMA Resources Operator
+
+NUMA Resources Operator deploys resources that allows you to schedule NUMA-aware workloads and deployments. You can install the NUMA Resources Operator using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== Installing the NUMA Resources Operator using the CLI
+
+As a cluster administrator, you can install the Operator using the CLI.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-numaresources-operator
+----
+
+.. Create the `Namespace` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-namespace.yaml
+----
+
+. Create a operator group for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-numaresources-operator
+  namespace: openshift-numaresources-operator
+spec:
+  targetNamespaces:
+  - openshift-numaresources-operator
+----
+
+.. Create the `OperatorGroup` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-operatorgroup.yaml
+----
+
+. Create a subscription for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-sub.yaml` file:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: numaresources-operator
+  namespace: openshift-numaresources-operator
+spec:
+  channel: "latest-{product-version}"
+  name: numaresources-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-sub.yaml
+----
+
+.Verification
+
+Verify that the required resources deployed successfully:
+
+[source,terminal]
+----
+$ oc get all -n openshift-numaresources-operator
+----
+
+.Example output
```

Comment:
this is correct but only after the `numaresourcesoperator` and `numaresourcesscheduler` resources have been created and processed correctly. We need to add these examples I believe.